README.md

```diff
@@ -155,6 +155,22 @@ For instance, you can ignore all the npm modules:
 ignoreFilenames: ['node_modules'],
 ```
 
+### `additionalLibraries`
+
+This option gives the possibility to remove other `propTypes` in addition to the canonical `prop-types`.
+
+For instance, by default
+```js
+import PropTypes from 'prop-types'
+import ImmutablePropTypes from 'react-immutable-proptypes'
+```
+will result in `propTypes` not to be removed; with:
```

Comment:
with this PR both are removed if the option is used.

The default behaviour will be, however, that `prop-types` could be removed, but `.propTypes` still contains `ImmutablePropTypes` fields, so nothing is removed.
Just as the current behaviour.